src/init.py

```diff
@@ -0,0 +1,128 @@
+import glob
+import os
+import pathlib
+import shutil
+import subprocess
```

Comment:
```suggestion
import shutil
import subprocess
from pathlib import Path
```

Reply:
This has been implemented in latest version